code/API_definitions/qos-booking-and-assignment.yaml

```
@@ -723,14 +722,15 @@ components:
         - The size of the array should be less than or equal to the devices that are remaining in the original booking.
         - At least one device identifier is required when a 2-legged access token is used.
         - In a 3-legged situation, only one device can be assigned at a time, and the device details are automatically generated from the access token
-        - In such a situation, 'Devices' array SHOULD NOT be provided.
+        - In such a situation, `Devices` array SHOULD NOT be provided.
       type: object
       properties:
         devices:
           $ref: "#/components/schemas/Devices"
         sink:
           type: string
           format: uri
+          pattern: "^https:\/\/.+$"

```

The pattern `^https:\/\/.+$` should use no quotes (as in Commonalities) or single quotes, but not double quotes (with double quotes the `\` will be interpreted in YAML as escape, changing the regex.

Let's align with commonalities and keep this pattern unquoted:

```suggestion
          pattern: ^https:\/\/.+$
```